packages/mgt-chat/src/components/ChatListHeader/ChatListHeader.tsx

```diff
@@ -39,27 +50,34 @@ export const ChatListHeader = (
   props: MgtTemplateProps &
     IChatListMenuItemsProps & {
       buttonItems?: ChatListButtonItem[];
+      bannerMessage?: string;
```

Comment:
If this is ? anyway, why force the banner message to be an empty string instead of supporting undefined.

Reply:
i think it's better to keep the bannerMessage either an empty string which means to hide OR with text. this shows the intend better than having 3 states i.e. undefined, empty string and string with text. I will rework